qa/suites/rados/perf/workloads/radosbench_omap_write.yaml

```diff
@@ -0,0 +1,5 @@
+tasks:
+- radosbench:
+    clients: [client.0]
+    write-omap: True
+    time: 300
```

Comment:
could you add the io/object size explicitly, so it's clear here and doesn't change if the defaults change in the future

Reply:
we default to 65536, as per the task https://github.com/ceph/ceph/blob/master/qa/tasks/radosbench.py#L79

Reply:
also added `size` (op_size)